docs/web-ui-auto-update-issue.md

```diff
@@ -0,0 +1,24 @@
+# Web UI Auto-Update Issue
+There is a known issue for old installation users(v1.0), the UI will get updated automatically after restarting the converter service. Since v2.0 is not compatible with v1.0, the old installation users may have some problems using the updated UI. 
```

Comment:
Describe the exact issue the users will face.

Reply:
Updated. Thanks!